packages/shorebird_cli/lib/src/commands/run_command.dart

```diff
@@ -73,6 +74,13 @@ class RunCommand extends ShorebirdCommand
       logger.err(utf8.decode(event));
     });
 
+    stdin
```

Comment:
Is there any issue with just piping the input directly?

```dart
flutter.stdin.addStream(stdin);
```

Reply:
the `addStream()` returns a future and is a blocking process (requires await). I wasn't sure if blocking threads would be the right approach, hence took a non-blocking approach.

Reply:
I think we can wrap it in an unawaited. It seems unnecessary to transform and decode everything imo.

Reply:
Makes sense! I have updated the commit!